samples/app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "gandi.translate-sample",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "react-scripts": "0.7.0"
+  },
+  "dependencies": {
+    "@gandi/react-translate": "^1.0.0-alpha",
+    "colors": "^1.1.2",
+    "lodash": "^4.17.2",
+    "moment": "^2.17.0",
+    "react": "^15.4.1",
+    "react-dom": "^15.4.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
```

Comment:
you should add `"i18n": "react-translate-scripts"` in order to launch extract, merge, update catlogs scripts